Makefile

```diff
@@ -58,6 +58,21 @@ production: paas
 	$(eval AZURE_BACKUP_STORAGE_ACCOUNT_NAME=s165p01dqtapidbbackup)
 	$(eval AZURE_BACKUP_STORAGE_CONTAINER_NAME=dqt-api)
 
+.PHONY: dv_review
+dv_review: aks dev-cluster
+	$(if $(CLUSTER), , $(error Missing environment variable "CLUSTER", Please specify a dev cluster name (eg 'cluster1')))
+	$(if $(IMAGE), , $(error Missing environment variable "IMAGE", Please specify an image tag for your review app))
```

Comment:
are we just creating a single env? otherwise it should have a pr-number or app-name if it's for multiple review type apps.

Reply:
Added APP_NAME and revlavent terraform changes